tests/helpers/perf_results_helper.rb

```diff
@@ -359,15 +377,23 @@ def average_csv(data_csv_path, skip_first_column = false)
   def validate_csv(csv_path)
     raise "File not found: #{csv_path}" unless File.exist?(csv_path)
 
-    validator = Csvlint::Validator.new(File.new(csv_path))
+    raise "Not a CSV file: #{csv_path}" unless File.extname(csv_path).eql?(".csv")
```

Comment:
A file extension is a poor indicator of an adherence to the validation.  Is this line necessary?  It seems that the check in the body of this method should address the validation.

Reply:
A file being validated by this method should not be considered valid unless it has a '.csv' extension, even if it is a valid CSV file. It is not the sole determining factor for validation but it is a basic requirement for CSV files generated and processed by the tool and for inclusion in the test results.